index.js

```diff
@@ -161,6 +161,13 @@ async function handleMessage(msg, ws) {
                 return errorMsg;
             }
             break
+        case "liquidity2":
```

Comment:
I'd change one thing here, and that's to change the name of the request message to `refreshliquidity2`, so there's a distinction between it and the response message